tests/queries/0_stateless/01710_minmax_count_projection.sql

```diff
@@ -16,7 +16,7 @@ select min(i), max(i), count() from d where _partition_value.1 = 10 group by _pa
 select min(i) from d where 1 = _partition_value.1;
 
 -- fuzz crash https://github.com/ClickHouse/ClickHouse/issues/37151
-SELECT min(i), max(i), count() FROM d WHERE (_partition_value.1) = 0 GROUP BY ignore(bitTest(ignore(NULL), 65535), NULL, (_partition_value.1) = 7, '10.25', bitTest(NULL, -9223372036854775808), NULL, ignore(ignore(-2147483647, NULL)), 1024), _partition_id ORDER BY _partition_id ASC NULLS FIRST;
+SELECT min(i), max(i), count() FROM d WHERE (_partition_value.1) = 0 GROUP BY ignore(bitTest(ignore(NULL), 0), NULL, (_partition_value.1) = 7, '10.25', bitTest(NULL, 0), NULL, ignore(ignore(-2147483647, NULL)), 1024), _partition_id ORDER BY _partition_id ASC NULLS FIRST;
```

Comment:
While at it, you probably want to fix the docs, it says `The countdown starts from 0 from the right to the left.`, it should say: `Counting is right-to-left, starting at 0.`